libraries/matrixui/src/main/kotlin/io/element/android/libraries/matrix/ui/messages/ToPlainText.kt

```diff
@@ -29,50 +32,87 @@ import org.jsoup.select.NodeVisitor
  * Converts the HTML string in [TextMessageType.formatted] to a plain text representation by parsing it and removing all formatting.
  * If the message is not formatted or the format is not [MessageFormat.HTML], the [TextMessageType.body] is returned instead.
  */
-fun TextMessageType.toPlainText() = formatted?.toPlainText() ?: body
+fun TextMessageType.toPlainText(stringProvider: StringProvider? = null) =
```

Comment:
Wouldn't it be better to always pass a non-null `StringProvider`?

Reply:
I tested it, but it's quite painful since this method is used in data class constructor, and other place where we do not have a stringProvider.

Reply:
Actually my bad, it's only he case for `Document.toPlainText`. I will give it a try.

Reply:
So I have tried it and it's adding lots of noise (Let me know if you want to see the code). Since there is no added value, I think we can keep the parameter optional. Do you agree @jmartinesp ?